std/string.d

```diff
@@ -500,6 +541,24 @@ ptrdiff_t indexOf(Range)(auto ref Range s, in dchar c,
     return indexOf!(StringTypeOf!Range)(s, c, cs);
 }
 
+///
```

Comment:
Surprised to see that having the documented unittests after an undocumented overload works.